src/System.Private.Xml/src/System/Xml/Xsl/Xslt/XPathPatternParser.cs

```diff
@@ -164,7 +164,7 @@ private QilNode ParseRelativePathPattern()
         {
             if (++_parseRelativePath > MaxParseRelativePathDepth)
             {
-                if (LocalAppContextSwitches.LimitXPathComplexity)
```

Comment:
@AlexGhiondea I'm not sure if I got what you meant. This `LocalAppContextSwitches` is a class of few properties that we created **internal** to System.Private.Xml as a workaround for the context switches when we ported System.Xml from Desktop. I think in this case, the name doesn't matter?